src/internal/storeSubscribable.ts

```diff
@@ -0,0 +1,40 @@
+import type {
+  SubscribableStore,
+  SubscriberFunction,
+  SubscriberObject,
+  UnsubscribeFunction,
+} from '../types';
+import { RawStoreFlags } from './store';
+import { RawStoreTrackingUsage } from './storeTrackingUsage';
+import { normalizeUnsubscribe } from './unsubscribe';
+
+export class RawSubscribableWrapper<T> extends RawStoreTrackingUsage<T> {
+  subscriber: Pick<SubscriberObject<T>, 'next'> & SubscriberFunction<T> = this.createSubscriber();
+  unsubscribe: UnsubscribeFunction | null = null;
+  override flags = RawStoreFlags.HAS_VISIBLE_ONUSE;
+
+  constructor(public subscribable: SubscribableStore<T>) {
+    super(undefined as any);
+  }
+
+  private createSubscriber() {
+    const subscriber = (value: T) => this.set(value);
+    subscriber.next = subscriber;
+    subscriber.pause = () => {
+      this.markConsumersDirty();
+    };
+    return subscriber;
+  }
+
+  override startUse(): void {
+    this.unsubscribe = normalizeUnsubscribe(this.subscribable.subscribe(this.subscriber));
+  }
+
+  override endUse(): void {
+    const unsubscribe = this.unsubscribe;
```

Comment:
I have not tested the speed of this, but I think it is better for correctness, in case unsubscribe triggers something that resubscribes (maybe this odd edge case should be tested ?).